tests/js/realm-tests.js

```diff
@@ -117,6 +117,16 @@ module.exports = BaseTest.extend({
         TestCase.assertEqual(Realm.defaultPath, newPath, "defaultPath should have been updated");
     },
 
+    testRealmSchemaVersion: function() {
+        TestCase.assertEqual(Realm.schemaVersion(Realm.defaultPath), 0xFFFFFFFFFFFFFFFF);
```

Comment:
I don't think anything is stored internally for a Realm that doesn't exist. The other bindings have 64-bit integers, and JS doesn't really. I think `-1` perfectly reflects a `does not exist` state (like with `indexOf`), whereas `0xFFFFFFFFFFFFFFFF` is just totally foreign when it comes to JS APIs.